client-app/src/examples/filemanager/AppModel.ts

```diff
@@ -1,18 +1,5 @@
-import {HoistAppModel, XH} from '@xh/hoist/core';
-import {OauthService} from '../../core/svc/OauthService';
+import {BaseAppModel} from '../../BaseAppModel';
 
-export class AppModel extends HoistAppModel {
+export class AppModel extends BaseAppModel {
     static instance: AppModel;
-
-    static override async preAuthAsync() {
-        await XH.installServicesAsync(OauthService);
-    }
-
-    override async logoutAsync() {
-        await XH.oauthService.logoutAsync();
-    }
-
-    override get supportsVersionBar(): boolean {
-        return window.self === window.top;
-    }
```

Comment:
Great that you added the BaseAppModel - nice to see this duplicated code go away.